src/site/content/en/progressive-web-apps/pwa-checklist/index.md

```diff
@@ -0,0 +1,482 @@
+---
+layout: post
+title: What makes a good Progressive Web App?
+authors:
+  - samrichard
+  - petelepage
+date: 2020-01-06
+updated: 2020-01-06
+description: |
+  What makes a good, or great Progressive Web App?
+tags:
+  - post
+  - progressive-web-apps
+draft: true
+---
+
+<!-- Disable heading-increment because it mucks with the Details widget -->
+<!--lint disable heading-increment-->
+
+Progressive Web Apps (PWA) are built and enhanced with modern APIs to deliver
+native-like capabilities, reliability, and installability while reaching
+_anyone, anywhere, on any device_ with a single codebase. To help you create
+the best possible experience, use the [core](#core) and [optimal](#optimal)
+checklists and recommendations to guide you.
+
+## Core Progressive Web App checklist {: #core }
+
+Because PWAs span all devices, from mobile through desktop, the core
+Progressive Web App Checklist is all about what you need to do to make your
+app installable and usable by all users, regardless of screen size or input type.
```

Comment:
Change sentence to:

The Progressive Web App Checklist describes what makes an app installable and usable by all users, regardless of size or input type.

It says the same thing a bit more directly and with few words. Also, 'describes' is a stronger verb than 'is'.